src/cli/make-description.js

```diff
@@ -1,7 +1,7 @@
-const chalk = require("chalk");
-const indentString = require("indent-string");
-const wrapAnsi = require("wrap-ansi");
-const semver = require("semver");
+import chalk from "chalk";
```

Comment:
Maybe make a dc rule so the esm only external reps don’t accidentally go in the part that we transpire down to commonjs.